### PR TITLE
Fix /oauth-redirect is mistakenly handled as 404

### PR DESCRIPTION
**Pull request by louischan-oursky**
ref DEV-2323

### CODE REVIEW

**Comment by louischan-oursky** [comment]
@tung2744 I discovered another problem. Do not merge yet.

**Comment by louischan-oursky** [comment]
Added one more commit to correct authui vite configuration.

File: authui/packages/vite-plugin-authgear-authui/src/index.js [hunk 1]
```diff
@@ -342,11 +366,6 @@ function buildPlugin({ input }) {
                 }
                 return null;
               },
-              hashCharacters: "hex",
-              assetFileNames: () => "[name].[hash][extname]",
-              chunkFileNames: () => "[name].[hash].js",
-              entryFileNames: () => "[name].[hash].js",
-              sourcemapFileNames: () => "[name].[chunkhash].map.js",
```
Comment (tung2744) [inline authui/packages/vite-plugin-authgear-authui/src/index.js:349]: Another thought: prefix the hash with some custom prefix like `aghash-[hash]` which we will check in the regex?

**Review by tung2744 — APPROVED** [review]

File: authui/packages/vite-plugin-authgear-authui/src/index.js (hunk 1 above)
Comment (louischan-oursky) [reply authui/packages/vite-plugin-authgear-authui/src/index.js:349]: Actually I want to unset `assetsDir: ""` to use the original behavior of Vite. Then the generated built assets should reside in a separate directory. We do not need the regex at all.

**Pull request merged**